day1/day1.py

```diff
@@ -0,0 +1,39 @@
+def list_changes(data):
+    changes = []
+    for line in data:
+        line = line.strip()
+        if (line[0] == '+'):
+            changes.append(int(line[1:]))
+        else:
+            changes.append( - int(line[1:]))
+    return changes
+
+
+def first_repeat(data):
+    changes = list_changes(data)
+    curr_freq = 0
+    i = 0
+    freqs = []
+    while True:
+        next_item = changes[i % len(changes)]
```

Comment:
I like that trick with reminder. In python, you can also leverage https://docs.python.org/2/library/itertools.html#itertools.cycle